components/kucoin_futures/actions/generic-api-call/generic-api-call.mjs

```diff
@@ -0,0 +1,51 @@
+import kucoin_futures from "../../kucoin_futures.app.mjs";
+
+export default {
+  name: "Kucoin Futures Generic API Call",
+  version: "0.0.1",
+  key: "kucoin_futures-generic-api-call",
+  description: "Make any API call for Kucoin Futures as per the documentation." +
+        "[reference](https://docs.kucoin.com/futures/#general)",
```

Comment:
We normally use the following pattern on description: 
`Make any API call for Kucoin Futures as per the documentation. See docs [here](https://docs.kucoin.com/futures/#general)`